samples/Elmah.Io.NLog.AspNetCore/NLog.config

```diff
@@ -10,7 +10,7 @@
   </extensions>
 
   <targets>
-    <target name="elmahio" type="elmah.io" apiKey="API_KEY" logId="LOG_ID" />
+    <target name="elmahio" type="elmah.io" apiKey="API_KEY" logId="LOG_ID" application="APP_NAME" />
```

Comment:
Maybe add another commented out target below, that show the usage of `application`. Would like the sample config to be as simple as possible.